scripts/index.js

```diff
@@ -1,25 +1,41 @@
+// worked with Eric M.
 
 // Implement the following functions to fulfill the tests!
-function getLocationName() {}
-function getLocationCountry() {}
-function getLocationLatitude() {}
-function getLocationLongitude() {}
-function getDescription() {}
-function getWindSpeed() {}
-function getSunrise() {}
+function getLocationName(weather) {
+  const city = new Map();
+  const cityValues = Object.values(weather); // array of city values
```

Comment:
Interesting approach! But, be wary of the fact that the order may not be reliable. Try to retrieve values by their key name when working with objects/dictionaries/maps.